apps/web/pages/signup.tsx

```diff
@@ -0,0 +1,187 @@
+// code duplicate of apps/api/auth/signup.tsx, could not find a way to import without breaking SSR
```

Comment:
when importing from /api/
<img width="578" alt="CleanShot 2022-10-12 at 09 53 11@2x" src="https://user-images.githubusercontent.com/8019099/195297438-d1b77a93-8243-4b33-9404-96442011bf56.png">

we get this

<img width="1039" alt="CleanShot 2022-10-12 at 09 53 00@2x" src="https://user-images.githubusercontent.com/8019099/195297412-e7a4c6eb-7b70-46ee-ac36-416b26957f53.png">

Reply:
likely because /api is a private submodule?